server/transport.go

```diff
@@ -24,13 +24,13 @@ func MakeHTTPHandler(e *Endpoints, svc Service, logger kitlog.Logger) http.Handl
 	}
 
 	r := mux.NewRouter()
-	r.Methods("GET").Path("/scep").Handler(kithttp.NewServer(
```

Comment:
Any reason we needed to remove that path?

Reply:
I think this includes some commits from the branch we're using. IIRC this is so we can customize the path

Reply:
yeah, I realized that as well. Should I change the target to be the `remove-path-setting-on-scep-handler` branch instead @roperzh ?

Reply:
@jahzielv jinx! I just left a review comment to coordinate on that. I think whatever works best for you both

Reply:
Yeah I think it makes sense to target `remove-path-setting-on-scep-handler`, will make it clearer what are the changes and what we bring into fleet. Note that after this PR lands, `go.mod` needs to be updated in fleet so that it actually uses those changes (the `replace` directive that indicates what commit to use: https://github.com/fleetdm/fleet/blob/main/go.mod#L332).

Reply:
Makes sense, and yep! I can file a PR in `fleet` to update the `go.mod` once this PR lands.